release/nightly_tests/dataset/pipelined_ingestion_compute.yaml

```diff
@@ -0,0 +1,30 @@
+cloud_id: cld_17WvYIBBkdgLwEUNcLeRAE
+region: us-west-2
+
+max_workers: 999
+
+aws:
+    IamInstanceProfile: {"Name": "ray-autoscaler-v1"}
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
+
+head_node_type:
+    name: head_node
+    instance_type: i3.8xlarge
+
+worker_node_types:
+    - name: memory_node
+      instance_type: i3.8xlarge
+      min_workers: 70
```

Comment:
Could be pretty expensive. How about something like 20 nodes to start? Hopefully that's 80% of the benefits of the test?